src/Pages/ProductDetails/Components/RelatedProducts/RelatedProduct.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import './RelatedProduct.scss';
+
+class RelatedProduct extends Component {
+  constructor() {
+    super();
+    this.state = {
+      x: 0,
+      sliderArr: [],
+      info: {},
+    };
+  }
+
+  goLeft = () => {
+    const { x, sliderArr } = this.state;
+    x === 0 ? this.setState({ x: 0 }) : this.setState({ x: x + 100 });
+  };
+
+  goRight = () => {
+    const { x, sliderArr } = this.state;
+    console.log(sliderArr);
+    x === -100 * (sliderArr.length - 1)
+      ? this.setState({ x: 0 })
+      : this.setState({ x: x - 100 });
+  };
+
+  componentDidMount() {
+    fetch('`http:// 10.58.6.70:8000/products/${this.props.match.params.id')
```

Comment:
```suggestion
    fetch(`${API}/products/${this.props.match.params.id}`)
```
- api 주소는 별도 분리해서 관리해주세요!
- 템플릿 리터럴(백틱)로 잘 감싸주세요! 지금은 url에 공백도 조재하고 제대로 작동하지 않을 것 같습니다.